painttiming.bs

```diff
@@ -62,9 +87,38 @@ Formally, we consider the user agent to have "rendered" a document when it has p
 
     NOTE: The rendering pipeline is very complex, and the timestamp should be the latest timestamp the user agent is able to note in this pipeline (best effort). Typically the time at which the frame is submitted to the OS for display is recommended for this API.
 
+An [=element=] is <dfn export>contentful</dfn> when one or more of the following apply:
+* The element has a [=text node=] child, with at least one character, excluding [=document white space characters=].
+* The element is a [=replaced element=] representing a [=loaded=] image.
+* The element's [=used=] style contains a [=loaded=] [=background-image=].
+* The element is a [=canvas=] with its [=context mode=] set to any value other than <code>none</code>.
+* The element is an [=svg element with rendered descendants=].
+* The element is an [=originating element=] for a [=paintable pseudo-element=] that represents a [=loaded=] image or text with at least one character excluding [=document white space characters=].
```

Comment:
It seems weird to require the 'paintable' part here since we're defining contentful. I think this should just say pseudo-element.

Reply:
But I'm defining paintable pseudo-element in this spec, it's not the same as pseudo-element.
It's maybe confusing but it's accurate that an element might be contentful if it contains a paintable pseudo-element.

Reply:
In any case, I reordered the definitions.